### DISCUSSION TITLE
`no-unsafe-return`, https://typescript-eslint.io/rules/no-unsafe-return/

### DISCUSSION THREAD

**Discussion by jpike88** [biomejs/biome#590] [Ideas]
This one is very handy, just ran into a nasty bug today where a type any is coerced to an interface type without some extra properties that were added later.

**Comment by ematipico** [comment]
Thank you @jpike88 for the suggestion. We have https://github.com/biomejs/biome/discussions/3 to collect rules from other sources. Would you mind add a comment there instead?
